test/groovy/com/sap/piper/BashUtilsTest.groovy

```diff
@@ -0,0 +1,47 @@
+package com.sap.piper
+
+import org.junit.Assert
+import org.junit.Test
+
+class BashUtilsTest {
+    @Test
+    void escapeFilePath() {
+        // Given: A Windows-style file path C:\some\path
```

Comment:
[curiosity] why do we take windows like pathes into account?

Reply:
Just to test edge cases